packages/web/src/scss/tools/_links.scss

```
@@ -1,20 +1,20 @@
-@use '@tokens' as tokens;
+@use '@global' as global-tokens;
 @use '../settings/links';
 
 @mixin base($set-color: false) {
     text-decoration-skip-ink: links.$text-decoration-skip-ink;
     text-underline-offset: links.$text-underline-offset;
 
     @if $set-color {
-        color: tokens.$action-link-primary-default;
+        color: var(--spirit-color-action-link-primary-default);
```

This could be read from `scss/settings/_global.scss`.

```suggestion
        color: var(--#{global.$prefix}color-action-link-primary-default);
```